src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml

```diff
@@ -0,0 +1,244 @@
+      attribute: "CurrentLevel"
+      response:
+          value: MinLevelValue
+
+    - label:
+          "Step 3: TH sends a StoreScene command to DUT with the GroupID field
+          set to G1 and the SceneID field set to 0x01."
+      command: "StoreScene"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: G1
+              - name: "SceneID"
+                value: 0x01
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
+              - name: "GroupID"
+                value: G1
+              - name: "SceneID"
+                value: 0x01
+
+    - label: "Step 4: TH sends a AddScene command to DUT with the GroupID field
+          set to G1, the SceneID field set to 0x02, the TransitionTime field set
+          to 0 and the ExtensionFieldSets set to: '[{ ClusterID: 0x0008,
+          AttributeValueList: [{ AttributeID: 0x0000, ValueUnsigned8: 0x64 }]}]'
+          "
+      command: "AddScene"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: G1
+              - name: "SceneID"
+                value: 0x02
+              - name: "TransitionTime"
+                value: 0
+              - name: "SceneName"
+                value: "Scene1"
+              - name: "ExtensionFieldSets"
+                value:
+                    [
+                        {
+                            ClusterID: 0x0008,
+                            AttributeValueList:
+                                [{ AttributeID: 0x0000, ValueUnsigned8: 0x64 }],
+                        },
+                    ]
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
+              - name: "GroupID"
+                value: G1
+              - name: "SceneID"
+                value: 0x02
+
+    - label:
+          "Step 5a: TH sends a RecallScene command to DUT with the GroupID field
+          set to G1 and the SceneID field set to 0x02."
+      command: "RecallScene"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: G1
+              - name: "SceneID"
+                value: 0x02
+
+    - label: "Step 5b: TH reads the CurrentLevel attribute from DUT"
+      cluster: "Level Control"
+      command: "readAttribute"
+      attribute: "CurrentLevel"
+      response:
+          value: 0x64
+
+    - label:
+          "Step 6a: TH sends a RecallScene command to DUT with the GroupID field
+          set to G1 and the SceneID field set to 0x01."
+      command: "RecallScene"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: G1
+              - name: "SceneID"
+                value: 0x01
+
+    - label: "Step 6c: TH reads the CurrentLevel attribute from DUT"
+      cluster: "Level Control"
+      command: "readAttribute"
+      attribute: "CurrentLevel"
+      response:
+          value: MinLevelValue
```

Comment:
I know it's not part of the test plan, but you might want to consider adding a remove all scenes and remove all groups at the end so the device is left in the expected state.